packages/frontend/src/components/ExecutionStep/hooks/useExecutionStepStatus.ts

```diff
@@ -63,6 +64,22 @@ export function useExecutionStepStatus({
     return failureIcon
   }, [isPartialSuccess, isStepSuccessful, status])
 
+  const customRetryButtonText = useMemo(() => {
+    // specific to delay until action where we want to allow users to manually
+    // retry and bypass the errors:
+    // - Invalid timestamp entered
+    // - Delay until timestamp entered is in the past
+    if (
+      [
+        'Invalid timestamp entered',
+        'Delay until timestamp entered is in the past',
+      ].includes(errorDetails?.name)
+    ) {
+      return 'Retry and send now'
+    }
```

Comment:
let's try to reduce the number of custom `ifs` here. Just return in the error message from the backend. 

`This action was scheduled to run at a time that has already passed. Click "Retry" to skip the delay and continue the pipe now.`